.github/workflows/workflow.yml

```diff
@@ -7,91 +7,160 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - name: "1. Clone repository"
+        uses: actions/checkout@v2
 
-      - name: Set up Python 3.6
+      - name: "2. Set up Python 3.6"
         uses: actions/setup-python@v1
         with:
           python-version: 3.6
 
-      - name: Prepare machine
+      - name: "3. Update pip"
         run: |
           python -m pip install --upgrade pip
 
-      - uses: actions/cache@v1
+      - name: "4. Attache cache for pip"
+        uses: actions/cache@v1
         id: cache
         with:
           path: ~/.cache/pip
           key: tests-${{ runner.os }}-pip-${{ hashFiles('requirements/main.txt', 'requirements/dev.txt') }}
           restore-keys: |
             tests-${{ runner.os }}-pip-
 
-      - name: Install dependencies
+      - name: "5. Install main and dev dependencies"
         run: |
           pip install -q .
```

Comment:
don't see requirements/main.txt being specified here

Reply:
setup.py (`so pip install .`) installs the main reqs